packages/api-derive/src/elections/approvalsOfAt.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright 2017-2019 @polkadot/api-derive authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { ApiInterfaceRx } from '@polkadot/api/types';
+import { Observable } from 'rxjs';
+import { AccountId, Vector, SetIndex, ApprovalFlag } from '@polkadot/types';
+import { map } from 'rxjs/operators';
+import { drr } from '../util/drr';
+
+export function approvalFlagsToBools (flags: Vector<ApprovalFlag>): boolean[] {
```

Comment:
Rather move this `util` - there is one in there already, I suppose this can also be added to the same file.